network-api/networkapi/wagtailpages/models.py

```diff
@@ -608,19 +656,29 @@ def generate_entries_set_html(self, request, *args, **kwargs):
         entries = self.get_entries()
         has_next = end < len(entries)
 
+        hide_classifiers = False
```

Comment:
Wondering why we need to set `hide_classifiers` here. Aren't they already taken care of in `filter_entries_for_tag` / `filter_entries_for_category`?

Reply:
No: the template will show the first tag/all tags, and for categories we need to show "nothing". The filter functions only filter which entries to show, they don't do anything with the data for those entries. That's the template's job.

Reply:
specifically, when we get here from `generate_category_entries_set_html`, we get all entries using `entries = self.get_entries()`, which lacks a predefined context variable. Instead, we need to _build_ a context for use with the `render()` call below